README.md

```diff
@@ -11,14 +11,23 @@ The old version (deprecated) is still available in the `legacy` branch.
 
 This is a very opinionated project, as I am a Tech Lead, this theme is very developer-focused.
 
+## Requirements
+
+### Nerd Fonts
+
+This theme requires the use of a patched font with Nerd Font. Ensure your terminal is set to use one before installing this theme. We recommend using FiraCode Nerd Font for this theme, but any patched font will do.
+
+### Noto Fonts
```

Comment:
Noto is not required, in fact in my system I removed it in order to have MS Emoji font to work 
![image](https://github.com/janoamaral/tokyo-night-tmux/assets/10008708/bbe23fe0-bd6d-4d37-875a-c167c558720d)

Reply:
Did you try with the following command:

```bash
fc-list | grep -i noto
```

?

Reply:
Maybe it's in Windows 3D emoji font, but afaik you only need a Nerd font. The theme uses pure unicode or icon.

![image](https://github.com/janoamaral/tokyo-night-tmux/assets/10008708/d326cb11-4ef3-4e61-b9d3-99613ba80cde)